### DISCUSSION TITLE
Vote for xxchan as RisingLight member

### DISCUSSION THREAD

**Discussion by st1page** [risinglightdb/risinglight#460] [Member]
@xxchan has made great contributions to the RisingLight's compute engine in the past month. Here's a list of their pull requests: https://github.com/risinglightdb/risinglight/pulls?q=author%3Axxchan

In #440, @xxchan add human-readable messages for create & insert & explain SQL statment. 

In #443, @xxchan support multi columns as hash key in the hash join executor and refactor the related codes in optimizer, which make risinglight run complex queries( such as TPC-H q5) with a better plan.

In #446 , @xxchan implement the TopN executor and add the rule converting limit-orderby to topN in optimizer, which improve the performance of the query with limit and order by clause. Also this PR fix the bug #404

@xxchan demonstrated a good understanding of the Rust programming language and the RisingLight project. Therefore, I'd like to promote them as RisingLight member.

If you are a member of the RisingLight organization, you may vote for this proposal with 👍 and 👎. According to [GOVERNANCE](https://github.com/risinglightdb/risinglight/blob/main/GOVERNANCE.md), a minimum requirement of 2 votes are required for accepting this proposal. Thanks!

**Comment by skyzh** [comment]
With 7 votes from @st1page @skyzh @TennyZhuang @likg227 @zzl200012 @ludics @wangrunji0408, we are accepting this vote. Welcome to the RisingLight community! The invitation email should have been sent. And @xxchan will be granted write access to all our repos.

**Comment by xxchan** [comment]
Many thanks! 🥰
